broken-site-reporting/README.md

```diff
@@ -25,6 +25,8 @@ Test suite specific fields:
 - `model` - string - name of the device model (native apps only)
 - `os` - string - operating system name (native apps only)
 - `gpcEnabled` - boolean - if GPC is enabled or not (native apps only) - GPC can be disabled by user or by remote config
+- `errorDescription` - string - optional field containing a description from an error generated by failed page load
+- `httpStatusCode` - int - optional field containing the HTTP status code returned by the page load
```

Comment:
There can be multiple codes, so make this `httpStatusCodes` and an array

E.g., for an array [304, 200], on Firefox we'd see this sent as `304%2C200` (can you add that as a test case please?)

If we want to keep the naming consistent, we could use `httpErrorCodes` here